pettingzoo/atari/base_atari_env.py

```diff
@@ -256,10 +271,8 @@ def render(self, mode="human"):
             self._screen.blit(myImage, (0, 0))
 
             pygame.display.flip()
-        elif mode == "rgb_array":
+        else:  # self.render_mode == "rgb_array":
```

Comment:
Should the `rgb_array` conditional be here? Seems like that mode is not being accounted for unless I'm missing someething.

Reply:
Right now, gym.make checks that render_mode is in `metadata['render_modes']` (if it is not None), thus they should be equivalent.
Anyway, for clarity it could be better to keep what you suggested or to put an assert, what do you think?

Reply:
Yes, but isn't this conditional in `def render()`? If that's the case, shouldn't it return the image only if `rgb_array` is triggered? Right now it seems that `image` is only returned if render mode is `human`, otherwise it does nothing. Unless I'm not seeing the code properly that is, which is entirely possible.

Reply:
Right now, the code displays the image in case of `human` mode otherwise returns the image.
Given the check performed by `gym.make`, this code should work equally (except if someone changes `render_mode` on the fly that should not be done, and in that case both code performs poorly).
Anyway, for clarity, I restored the previous code